super/pom.xml

```diff
@@ -472,6 +472,10 @@
             <groupId>org.springframework</groupId>
             <artifactId>spring-aop</artifactId>
           </exclusion>
+          <exclusion>
+            <groupId>com.querydsl</groupId>
+            <artifactId>querydsl-apt</artifactId>
+          </exclusion>
```

Comment:
Is this exclusion needed for getting the CycloneDC SBOM plugin to work?

Reply:
removed